[2024-10-09T21:34:21.839+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-09T21:34:21.877+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LoadWebScraping.copy_data_to_postgres manual__2024-10-09T21:34:17.501052+00:00 [queued]>
[2024-10-09T21:34:21.895+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LoadWebScraping.copy_data_to_postgres manual__2024-10-09T21:34:17.501052+00:00 [queued]>
[2024-10-09T21:34:21.896+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-09T21:34:21.914+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): copy_data_to_postgres> on 2024-10-09 21:34:17.501052+00:00
[2024-10-09T21:34:21.924+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'LoadWebScraping', 'copy_data_to_postgres', 'manual__2024-10-09T21:34:17.501052+00:00', '--job-id', '1504', '--raw', '--subdir', 'DAGS_FOLDER/LoadWebScraping.py', '--cfg-path', '/tmp/tmpljpkldl6']
[2024-10-09T21:34:21.928+0000] {standard_task_runner.py:105} INFO - Job 1504: Subtask copy_data_to_postgres
[2024-10-09T21:34:21.922+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6033) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-09T21:34:21.929+0000] {standard_task_runner.py:72} INFO - Started process 6035 to run task
[2024-10-09T21:34:21.978+0000] {task_command.py:467} INFO - Running <TaskInstance: LoadWebScraping.copy_data_to_postgres manual__2024-10-09T21:34:17.501052+00:00 [running]> on host 97b4e38b3eb9
[2024-10-09T21:34:22.260+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='LoadWebScraping' AIRFLOW_CTX_TASK_ID='copy_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-10-09T21:34:17.501052+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-09T21:34:17.501052+00:00'
[2024-10-09T21:34:22.262+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-09T21:34:22.282+0000] {sql.py:266} INFO - Executing: 
        COPY public.tb_dim_site FROM '/opt/***/datalake/silverzone/pichau_2024-10-08/dim_site.parquet'
        WITH (FORMAT PARQUET);
        
[2024-10-09T21:34:22.292+0000] {base.py:84} INFO - Retrieving connection 'postgres-***'
[2024-10-09T21:34:22.358+0000] {base.py:84} INFO - Retrieving connection 'postgres-***'
[2024-10-09T21:34:22.368+0000] {sql.py:509} INFO - Running statement: 
        COPY public.tb_dim_site FROM '/opt/***/datalake/silverzone/pichau_2024-10-08/dim_site.parquet'
        WITH (FORMAT PARQUET);
        , parameters: None
[2024-10-09T21:34:22.377+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 272, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 456, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 514, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.InvalidParameterValue: COPY format "parquet" not recognized
LINE 3:         WITH (FORMAT PARQUET);
                      ^

[2024-10-09T21:34:22.396+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=LoadWebScraping, task_id=copy_data_to_postgres, run_id=manual__2024-10-09T21:34:17.501052+00:00, execution_date=20241009T213417, start_date=20241009T213421, end_date=20241009T213422
[2024-10-09T21:34:22.413+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-09T21:34:22.414+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 1504 for task copy_data_to_postgres (COPY format "parquet" not recognized
LINE 3:         WITH (FORMAT PARQUET);
                      ^
; 6035)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 272, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 456, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 514, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.InvalidParameterValue: COPY format "parquet" not recognized
LINE 3:         WITH (FORMAT PARQUET);
                      ^

[2024-10-09T21:34:22.465+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-09T21:34:22.482+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-09T21:34:22.489+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
